TooltipDecorator/TooltipDecorator.js

```diff
@@ -39,10 +30,10 @@ const defaultConfig = {
 	 * resolution.
 	 *
 	 * @type {Number}
-	 * @default 24
+	 * @default 48
 	 * @memberof sandstone/TooltipDecorator.TooltipDecorator.defaultConfig
 	 */
-	screenEdgeKeepout: (24 + 24),
+	screenEdgeKeepout: (24 + 24), // Do NOT forget to check useTooltip's default prop value also.
```

Comment:
Why don't we remove the comment here and make a constant from `useTooltip` to share the value here?

Reply:
🆗